numpy/lib/function_base.py

```diff
@@ -4064,7 +4066,8 @@ def _quantile_ureduce_func(a, q, axis=None, out=None, overwrite_input=False,
         r = _lerp(x_below, x_above, weights_above, out=out)
 
     # if any slice contained a nan, then all results on that slice are also nan
-    if np.any(n):
+    if (n.ndim == 0 and n[0]  # Fast-case for 1D.
```

Comment:
I don't get this. If `n.ndim == 0` then `n[0]` raises `IndexError`.

Reply:
sorry, that should have been `ndim == 1`.